sdk/include/reactos/wine/debug.h

```diff
@@ -181,7 +181,7 @@ extern int wine_dbg_log( enum __wine_debug_class cls, struct __wine_debug_channe
                          const char *format, ... ) __WINE_PRINTF_ATTR(4,5);
 /* ReactOS compliant debug format */
 extern int ros_dbg_log( enum __wine_debug_class cls, struct __wine_debug_channel *ch, const char *file,
-                         const char *func, const int line, const char *format, ... ) __WINE_PRINTF_ATTR(6,7);
+                         const int line, const char *func, const char *format, ... ) __WINE_PRINTF_ATTR(6,7);
```

Comment:
Why reverse those?

Reply:
(module > ) file > line > func

Reply:
No, a function almost always has multiple lines.

Reply:
Good argument against; I will revert in a subsequent update.